docs/howto/db.md

```diff
@@ -0,0 +1,28 @@
+(db)=
+# How to use a SQLair database
```

Comment:
In this entire db HTG the language is that of reference. Please convert to HTG language. (You do that in the tx HTG -- "To achieve x, do y. For example:" PS In some cases (e.g., the next paragraph) that will also significantly improve clarity.

Reply:
Good point, changed the wording throughout